### ISSUE TITLE
For accessibility, please add a keyboard shortcut to eject all mp3 devices

### ISSUE THREAD

**Issue by gcapiel** [nightingale-media-player/nightingale-hacking#122]
A number critical Songbird functions can be accessed via keyboard shortcuts, especially after initial setup.  For accessibility, please add a keyboard shortcut to eject all mp3 devices.  Unfortunately, OS level eject does not work on OS X, so you have to rely on Songbird for eject.
##
